hibernate5/src/main/java/com/baeldung/hibernate/DatabaseMultitenantUtil.java

```diff
@@ -0,0 +1,14 @@
+package com.baeldung.hibernate;
+
+public class DatabaseMultitenantUtil extends HibernateMultiTenantUtil{
+    
+    
+    @Override
+    public String tenantUrl(String originalUrl, String tenant, String tenantStrategy) throws UnsupportedTenancyException {
```

Comment:
The third parameter does not appear to be necessary in either of the implementations.

Reply:
Done. Sorry i Missed that.